src/meta-srv/src/procedure/region_failover/deactivate_region.rs

```diff
@@ -87,6 +99,17 @@ impl DeactivateRegion {
                     }.fail();
                 };
                 if result {
+                    let inactive_node_manager =
+                        InactiveNodeManager::new(&ctx.leader_cached_kv_store);
+                    inactive_node_manager
+                        .deregister_inactive_region(
```

Comment:
If a response for closing the region is not received, metasrv needs to ensure that leasing of the corresponding node's region is permanently disabled. This is the only way to ensure that the failed region opens correctly on another node.

Reply:
By the way, I have switched to using in_memory to store this data. This means that as the leader switches, the data will be lost and the new leader needs to consider this situation.

Reply:
The reason for switching to `in_memory` storage is because it is considered that only the region being closed will be cached, while the regions in normal state will penetrate the cache and access remote storage. This is not a desirable situation.
I thought about it, and since the procedure is executed on the leader node, there is no need to persist the data.

Reply:
If a node is selected as a cadidate, its inactive status will be cleared. @MichaelScofield 
https://github.com/GreptimeTeam/greptimedb/pull/2004/commits/23dcb16847c33984ab1a24252cebc41e5cbc5aae